### PR TITLE
Add an arm-v6 build [WIP]

### PR DESCRIPTION
**Pull request by intrigus**
This is a mostly complete arm-v6 build.
There are still some rough edges, but it should mostly work.

@Spasi please review :)

### CODE REVIEW

File: modules/core/src/main/c/util/remotery/Remotery.c [hunk 1]
```diff
@@ -156,20 +156,7 @@ static rmtU8 minU8(rmtU8 a, rmtU8 b)
 {
     return a < b ? a : b;
 }
-static rmtU16 minU16(rmtU16 a, rmtU16 b)
```
Comment (intrigus) [inline modules/core/src/main/c/util/remotery/Remotery.c:159]: This can be ignored.

**Comment by Spasi** [comment]
Could you please squash all commits and rebase on top of current master (warning: there've been tons of changes)?

The changes look reasonable so far, except the custom docker image. Couldn't we use a standard image and install missing dependencies at build time?

Why is it a raspberry cross compiler and not a generic ARM cross compiler? Are there raspberry-specific stuff in there that will make the resulting binaries only work on raspberry?

Would ARMv7/ARMv8 builds look (very) different?

**Comment by intrigus** [comment]
@Spasi what do you consider a standard image? 

I used a raspberry cross compiler because it allowed me to easily install packages into the sysroot.
I think that the binaries should work on all armv6 cpus but I only tested it on my pi, so I am not sure. 

An ARMv7/ARMv8 build should not look different.

**Comment by Spasi** [comment]
> what do you consider a standard image?

Afaict, you have forked `sdt/docker-raspberry-pi-cross-compiler`, added missing dependencies and the LWJGL builds now depend on [intrigus/docker-raspberry-pi-cross-compiler](https://hub.docker.com/r/intrigus/docker-raspberry-pi-cross-compiler/). Instead of doing that, we could use [sdt4docker/raspberry-pi-cross-compiler](https://hub.docker.com/r/sdt4docker/raspberry-pi-cross-compiler/) directly and install dependencies using the .travis script (via `rpxc`).

The build will be a bit slower, but it would be easier to maintain. Only the .travis script will have to change to update to future dependencies, instead of building a new docker image and uploading it. Also, different libraries need different dependencies, we don't have to put everything inside the same image.

**Pull request closed without merging**